cli/cmd/root.go

```diff
@@ -34,7 +36,10 @@ func init() {
 	cobra.OnInitialize(initConfig)
 	cobra.EnablePrefixMatching = true
 
-	rootCmd.PersistentFlags().StringVar(&kubeconfig, "kubeconfig", "", "kubeconfig file (default is $HOME/.kube/config)")
+	rootCmd.PersistentFlags().StringVarP(&kubeconfig, "kubeconfig", "k", "", "kubeconfig file location (default is $HOME/.kube/config)")
+	rootCmd.PersistentFlags().StringP("namespace", "n", v1.NamespaceDefault, "Change namespace to modify objects in.")
```

Comment:
Maybe "select" instead of "change"? "Change" almost makes it sound like the setting will persist between calls to the cli. And mention that default is "default".